mysys/crc32/crc32_arm64.c

```diff
@@ -11,12 +11,13 @@
 #define HWCAP_CRC32 (1 << 7)
 #endif
 
-unsigned int crc32c_aarch64_available(void)
+/* ARM made crc32 default from ARMv8.1 but optional in ARMv8A
+so the runtime check. */
+unsigned int crc32_aarch64_available(void)
 {
 	unsigned long auxv = getauxval(AT_HWCAP);
 	return (auxv & HWCAP_CRC32) != 0;
 }
```

Comment:
To align with my suggestion for `my_sys.h`, the return type should be `int`. I think that it would be more future-proof to avoid declaring a local variable:
```c
int crc32_aarch64_available(void)
{
  return (getauxval(AT_HWCAP) & HWCAP_CRC32) != 0;
}
```

Reply:
renamed to int.